src/dagmc/CMakeLists.txt

```diff
@@ -1,48 +1,25 @@
-message(STATUS "Building DAGMC library")
+message("")
+
+file(GLOB SRC_FILES "*.cpp")
+file(GLOB PUB_HEADERS "*.hpp")
 
 # Configure version header
 configure_file(DagMCVersion.hpp.in DagMCVersion.hpp)
-list(APPEND DAGMC_PUB_HEADERS ${CMAKE_CURRENT_BINARY_DIR}/DagMCVersion.hpp)
-
-# Source files
-list(APPEND DAGMC_SRC_FILES DagMC.cpp)
-list(APPEND DAGMC_SRC_FILES dagmcmetadata.cpp)
-list(APPEND DAGMC_SRC_FILES ${CMAKE_SOURCE_DIR}/src/uwuw/uwuw.cpp)
-
-# Public headers
-list(APPEND DAGMC_PUB_HEADERS DagMC.hpp)
-list(APPEND DAGMC_PUB_HEADERS dagmcmetadata.hpp)
-list(APPEND DAGMC_PUB_HEADERS ${CMAKE_SOURCE_DIR}/src/uwuw/uwuw.hpp)
+list(APPEND PUB_HEADERS ${CMAKE_CURRENT_BINARY_DIR}/DagMCVersion.hpp)
 
-# Libraries
-set(LINK_LIBS)
-set(LINK_LIBS_STATIC)
-list(APPEND LINK_LIBS        ${DAG_LINK_LIBS})
-list(APPEND LINK_LIBS        pyne_dagmc)
-list(APPEND LINK_LIBS_STATIC ${DAG_LINK_LIBS})
-list(APPEND LINK_LIBS_STATIC pyne_dagmc-static)
+if (BUILD_STATIC_EXE)
+  set(LINK_LIBS lapack blas gfortran quadmath)
+else ()
+  set(LINK_LIBS)
+endif ()
+set(LINK_LIBS_EXTERN_NAMES MOAB_LIBRARIES HDF5_LIBRARIES)
 
-# Include directories
-include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-include_directories(${CMAKE_CURRENT_BINARY_DIR})
-include_directories(${HDF5_INCLUDE_DIRS})
```

Comment:
where did these wind up?

Reply:
pretty sure you never need to include the current source directory as it's automatically included. In other locations, `include_directories(${CMAKE_SOURCE_DIR}/src/dagmc)` is used. `include_directories(${CMAKE_BINARY_DIR}/src/dagmc)` is used here and as needed. HDF5 is included when finding MOAB.